builds/misc/images-release.yaml

```diff
@@ -312,6 +367,12 @@ jobs:
           name: "Diagnostics Module"
           project: IotedgeDiagnosticsDotnet
           version: $(version)
+      - template: templates/image-linux.yaml
+        parameters:
+          imageName: azureiotedge-metrics-collector
+          name: "Azure IoT Edge Metrics Collector"
+          project: Microsoft.Azure.Devices.Edge.Azure.Monitor
+          version: $(version)          
```

Comment:
@veyalla Are we publishing the metrics collector using the same version as all the other modules?

Reply:
I am wondering if we should follow the same versioning format as the api proxy?

Reply:
+1 to Andrew's comment. It should be versioned separately from system modules.